doc/api/http2.md

```diff
@@ -1854,6 +1854,14 @@ The given callback is registered as a listener on the `'timeout'` event.
 In case if `callback` is not a function, a new `ERR_INVALID_CALLBACK`
 error will be thrown.
 
+#### `server.updateSettings([settings])`
```

Comment:
This appears to be out of order? This entry should be after `server.timeout` below?